store/postgres/asset_repository.go

```diff
@@ -47,7 +47,6 @@ func (r *AssetRepository) GetAll(ctx context.Context, config asset.Config) (asse
 		return
 	}
 
-	assets = []asset.Asset{}
```

Comment:
oh actually this is to prevent `GetAll` to return `null` to client as opposed to empty array.

Reply:
I see, so I think we can make it simpler by not using `named parameter` in the return to make the flow clearer

Reply:
yeah sure, if it is bringing confusion the it is definitely needs to be refactored.